src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+import java.util.Scanner;
 public class Duke {
+
+    public static void drawLine(String symbol) {
+        for (int i = 0; i < 40; i++) {
+            System.out.print(symbol);
+        }
+        System.out.println();
+    }
+
+    public static void echo(Task[] tasks) {
+        drawLine("_");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  added: " + tasks[Task.getTaskCount() - 1].toString());
+        System.out.println("Now you have " + Task.getTaskCount() + " tasks in your list.");
+        drawLine("_");
+    }
+
+    public static void printTaskList(Task[] tasks) {
+        drawLine("_");
+        if (Task.getTaskCount() == 0) {
+            System.out.println("The list is empty...!");
+        }
+        for (int i = 0; i < Task.getTaskCount(); i++) {
```

Comment:
Good job in following the coding conventions by using "i" as an iterator variable